test/units/cli/galaxy/test_get_collection_widths.py

```diff
@@ -8,18 +8,16 @@
 import pytest
 
 from ansible.cli.galaxy import _get_collection_widths
+from ansible.galaxy.dependency_resolution.dataclasses import Requirement
 
 
 @pytest.fixture
 def collection_objects(mocker):
```

Comment:
mocker's unnecessary
```suggestion
def collection_objects():
    """Produce a list of 3 test collection requirements."""
```